context/app/static/js/components/home-revision/ImageCarouselContainer/ImageCarouselContainer.jsx

```diff
@@ -5,16 +5,19 @@ import ImageCarouselControlButtons from '../ImageCarouselControlButtons';
 import ImageCarouselCallToAction from '../ImageCarouselCallToAction';
 import { Flex, CallToActionWrapper } from './style';
 
+const numImages = 3;
```

Comment:
Can this be connected to the logic that controls the wrap around when you go past the last slide? Want to avoid having the same value repeated in multiple locations in the code.

Reply:
I think this is the only place it is defined. Maybe I'm misunderstanding?

Reply:
We pass this value to the control buttons as a prop.